assignments/4. operators/conditional-operator.md

```diff
@@ -1,62 +1,165 @@
 ## If Statement
+
 1.  🎖Make a simple calculator with these functions. Using prompt, type conversion, if else statement. Use prompt to take the input from user i.e two numbers and an operation (Add, Sub, Mul, Div).
 
-  ⛑ Rule
-    * [ ] While substracting and dividing keep in mind the number one should be greater then number two. If not show alert saying `Number Two is larger then Number one`.
-  ⚡️ Operations
-    * [ ] Add
-    * [ ] Sub
-    * [ ] Mul
-    * [ ] Div
+⛑ Rule
+_ [ ] While substracting and dividing keep in mind the number one should be greater then number two. If not show alert saying `Number Two is larger then Number one`.
+⚡️ Operations
+_ [ ] Add
+_ [ ] Sub
+_ [ ] Mul \* [ ] Div
 
 2. 🎖Write a if else statement which checks if the status is single `console.log` the message `John is single` or else `John is married`
+
 ```js
-var firstName = 'John';
-var status = 'single';
+var firstName = "John";
+var status = "single";
 // Your code goes here
 ```
 
+    var firstName = 'John';
+    var status = 'single';
+
+    if (status = 'single') {
```

Comment:
You are comparing the values so it should be `==` or `===` not `assignment operator` `=`.